src/styles/containers/SiteInfo.scss

```diff
@@ -33,75 +34,87 @@
 		}
 	}
 
-	.SiteInfo_Top {
-		padding: 30px;
+	.SiteInfo_Header {
 		position: relative;
```

Comment:
Curious, why was `position: relative` required here?

Reply:
Normally it would not make a difference. It only comes into play if an error banner pops up above the header section. Without a container with a relative position, the Start/Stop button will remain fixed and overlap the banner.